templates/usr/local/bin/rsync_debian_mirror.j2

```diff
@@ -73,6 +75,12 @@ function writetracefile() {
 
     mv "${base_path}/${trace}.new" "${base_path}/${trace}"
 
+    echo "$archive_version" >> ${base_path}/${trace}
+
+    if grep -q "pkg.adfinis.com" ${base_path}/project/trace/_traces; then
+           echo "pkg.adfinis.com" >> ${base_path}/project/trace/_traces
+    fi
```

Comment:
While unlikely that `$base_path` and `$trace` contain whitespace, I would still recommend quoting them to be safe.

Reply:
Resolved in 8db2c85f2ca988c73eb18ba1cb5f23e3d910209a